velox/exec/LocalPlanner.cpp

```diff
@@ -181,6 +182,11 @@ uint32_t maxDrivers(const DriverFactory& driverFactory) {
       if (!values->isParallelizable()) {
         return 1;
       }
+    } else if (
+        auto arrowStream =
```

Comment:
arrowStream variable is not used, hence, can be dropped

```
if (std::dynamic_pointer_cast<const core::ArrowStreamNode>(node))
```
```

Reply:
Fixed.